### PR TITLE
Updated Analytics API docs and added release note

### PR DESCRIPTION
**Pull request by suyashjoshi**

### CODE REVIEW

File: docs/analytics/aggregate.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
-# Using the Analytics APIs to access call performance aggregate data
+# Call Aggerate Reports
 
-## Composing a request to the Call Performance Aggregate API
+It is quite common to acquire and segment analytics data. Call Performance Aggregate API provides ability to aggregate call realted data in various ways such as by summing, averaging, getting minimum or maximum or percetage of data etc. For more information regarding common use cases, please refer to the [overview section](index.md) and for detailed information regarding how to compose API Request, please refer to the data guide below.
```
Comment (byrnereese) [inline docs/analytics/aggregate.md:3]: Please review this content for grammar and spelling. Install Grammarly or some other checker to ensure that sentence structure makes sense, and all words are spelled correctly.

File: docs/analytics/aggregate.md [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
 
-Please refer to our [API Reference](https://developers.ringcentral.com/api-reference/Line-Of-Business-Analytics/aggregatePerformanceReportCalls) for details about Aggregate API Endpoint.
+## Composing a request using RingCentral API Reference
+
+One can use our [interactive API Reference](https://developers.ringcentral.com/api-reference/Line-Of-Business-Analytics/aggregatePerformanceReportCalls) to not only create API Request but also run it interactively and see the response.
```
Comment (byrnereese) [inline docs/analytics/aggregate.md:7]: Please review for grammar

File: docs/analytics/timeline.md [hunk 3]
```diff
@@ -1,15 +1,20 @@
-# Using the Analytics API to assess performance during specific time periods
+# Call Timeline Reports
 
-## Composing a request to the Call Performance Timeline API
+It is quite common to acquire and segment analytics data based on time realted metrics. Call Performance Timeline API provides ability to segment call realted data in various ways such as by getting call duration based on response or actions etc. For more information regarding common use cases, please refer to the [overview section](index.md) and for detailed information regarding how to compose API Request, please refer to the data guide below.
 
-Please refer to our [API Reference](https://developers.ringcentral.com/api-reference/Line-Of-Business-Analytics/performanceReportCallsTimeline) for details about Timeline API Endpoint.
+## Composing a request using RingCentral API Reference
```
Comment (byrnereese) [inline docs/analytics/timeline.md:5]: Check for grammar. "Composing a request using the RingCentral..."

File: docs/analytics/timeline.md [hunk 4]
```diff
@@ -1,15 +1,20 @@
...
+
+One can use our [interactive API Reference](https://developers.ringcentral.com/api-reference/Line-Of-Business-Analytics/performanceReportCallsTimeline) to not only create API Request but also run it interactively and see the response.
```
Comment (byrnereese) [inline docs/analytics/timeline.md:7]: Grammar: to not only create **an** API request, but also run it interactively to see the response.

File: docs/basics/changelog/1_0_51.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
+# API Changelog for Apr 2022 (v.1.0.51)
+
+### Voice
+
+### Analytics
+
```
Comment (byrnereese) [inline docs/basics/changelog/1_0_51.md:6]: Do we need to keep these, or can they be removed?

File: docs/analytics/timeline.md (hunk 4 above)
Comment (suyashjoshi) [reply docs/analytics/timeline.md:7]: fixed

File: docs/analytics/aggregate.md (hunk 2 above)
Comment (suyashjoshi) [reply docs/analytics/aggregate.md:7]: ok

File: docs/analytics/aggregate.md (hunk 1 above)
Comment (suyashjoshi) [reply docs/analytics/aggregate.md:3]: ok

File: docs/basics/changelog/1_0_51.md (hunk 5 above)
Comment (suyashjoshi) [reply docs/basics/changelog/1_0_51.md:6]: removed 'Analytics' as it's also in subheading but kept 'Voice' as it's top-level category

**Comment by suyashjoshi** [comment]
Btw @CoCraig I've also incorporated your changes and sending new PR today.

**Comment by suyashjoshi** [comment]
Here is the link to review the changes (just merged them), I've incorporated all your feedback and fixed the page with 'grammar' issues - ran through grammarly, added key uses that @CoCraig asked so both API pages are well understood and also fixed formatting issues etc . https://github.com/ringcentral/ringcentral-api-docs/commit/89f2a9813a33d6c9ace790e38b468d533c8923f5

**Pull request merged**